ra/deployment/src/main/java/io/quarkus/artemis/jms/ra/deployment/health/ConnectorFactoryHealthCheckProcessor.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.artemis.jms.ra.deployment.health;
+
+import java.util.Optional;
+
+import io.quarkus.artemis.core.deployment.health.ArtemisHealthSupportBuildItem;
+import io.quarkus.artemis.jms.ra.runtime.ConnectionFactoryHealthCheck;
+import io.quarkus.deployment.Capabilities;
+import io.quarkus.deployment.Capability;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.smallrye.health.deployment.spi.HealthBuildItem;
+
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class ConnectorFactoryHealthCheckProcessor {
```

Comment:
This class seems useless as it does the same as what's in the other processor above?

Reply:
Ah different modules, just noticed that now